boards/DIM/Inc/App/states/App_DriveState.h

```diff
@@ -1,9 +1,26 @@
 #pragma once
 
 #include "App_SharedStateMachine.h"
+#include "App_SharedMacros.h"
+#include "App_SevenSegDisplays.h"
+#include "App_SharedExitCode.h"
 
 /**
  * Get a pointer to the Drive State.
  * @return A pointer to the Drive State. THIS SHOULD NOT BE MODIFIED.
  */
 const struct State *App_GetDriveState(void);
+
+/**
+ * Check CAN bus for BMS Fault messages
+ * @param can_rx Dim CAN RX interface for recieving messages
+ * @return True if any BMS Fault is present
+ */
+bool has_bms_fault(const struct DimCanRxInterface *can_rx);
+
```

Comment:
Let's stay consistent with naming conventions for now
App_DriveState_HasBmsFault

Reply:
Good call